linkatos/firebase.py

```diff
@@ -0,0 +1,33 @@
+import pyrebase
+
+
+def store_url(is_yes, url, FB_API_KEY, FB_USER, FB_PASS):
+    # connect to firebase
+    config = {
+        "apiKey": FB_API_KEY,
+        "authDomain": "coses-acbe6.firebaseapp.com",
+        "databaseURL": "https://coses-acbe6.firebaseio.com",
+        "storageBucket": "coses-acbe6.appspot.com"}
+
+    # do nothing if it's unnecessary
+    if is_yes is False:
```

Comment:
Do you expect `is_yes` to be something different to a boolean? If not, `if not is_yes:` reads more naturally.